src/scene/forward-renderer.js

```diff
@@ -1061,7 +1119,7 @@ Object.assign(ForwardRenderer.prototype, {
         var i;
         var point, spot;
 
-        var pnts = sortedLights[LIGHTTYPE_POINT];
+        var pnts = sortedLights[LIGHTTYPE_OMNI];
```

Comment:
maybe rename pnts to omnis

Reply:
renamed in all code except point light shadows - which are assuming punctual shape